docs/connect-kubectl.md

```diff
@@ -0,0 +1,48 @@
+# Connect kubectl
+
+In Kubernetes, the configuration for access to your cluster is a format known as `kubeconfig` that is normally stored as a file. It contains details such as cluster API server addresses and user access credentials. Treat it as sensitive data. Tools like `kubectl` use `kubeconfig` to connect and authenticate to a cluster and perform operations on it.
+Learn more about [kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) and [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) on [kubernetes.io](https://kubernetes.io).
+
+### Prerequisites
+- You are logged on to the Gardener Dashboard.
+- You have created a cluster and its status is operational.
+
+On this page:
+- [Downloading kubeconfig for a cluster](#downloading-kubeconfig-for-a-cluster)
+- [Connecting to the cluster](#connecting-to-the-cluster)
+- [Exporting KUBECONFIG environment variable](#exporting-kubeconfig-environment-variable)
+
+<br/>
+
+### Downloading kubeconfig for a cluster
+1. Select your project from the dropdown on the left, then choose **CLUSTERS** and locate your cluster in the list. Choose the *key* icon to bring up a dialog with the access options.
+
+   <img src="images/01-select-cluster.png">
```

Comment:
We need to be careful not to put too many images here. I know, images help to understand the context better, however it is a lot of effort to keep them up to date. Some features also may be hidden because of permissions or configuration. Moreover, we are about to introduce themes which will make the screenshots even less useful. I talked to the team, and we agreed to keep (most) of them for now, but maybe we will need to change things later.